pontoon/base/templatetags/helpers.py

```diff
@@ -231,3 +231,9 @@ def pretty_url(url):
         url = url[4:]
 
     return url
+
+
+@library.filter
+def local_url(url, code='en-US'):
```

Comment:
This is intentional. I'm trying to be consistent in using adjectives, we have `pretty_url` and `local_url`.